gustaf/_typing.py

```diff
@@ -35,19 +28,3 @@ def is_mesh(candidate: Any) -> bool:
         return isinstance(candidate, MESH_TYPES.__args__)
     else:
         return issubclass(type(candidate), MESH_TYPES)
-
-
-def is_spline(candidate: Any) -> bool:
```

Comment:
Could you take care of this in next PR? Was causing some import issues (cyclic import). btw, do you really need to keep all direct references to the class? this will be very difficult to stay away from cyclic import